src/Sarif/DefaultTraces.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    [Flags]
+    public enum DefaultTraces
```

Comment:
>DefaultTraces [](start = 16, length = 13)

Creating a new pattern here allowing for specific kinds of traces to be enabled on the command-line. Currently, the --verbose flag enables all this output, which is very copious.